src/frontend/parallelization_phase.nit

```diff
@@ -39,10 +39,9 @@ private class ParallelizationPhase
 			toolcontext.error(nmethdef.location, "Syntax error: only a method can be threaded.")
 			return
 		end
-		if nmethdef.n_signature.n_params.length != 0 then
-			toolcontext.error(nmethdef.location, "Syntax error: parametrized method not supported yet.")
-			return
-		end
+
+		#TODO: check for self calls
```

Comment:
the redirection to the original self would be hard to do in  nice way because self has a special status in some places (super-call, formal types, visibility) and even replacing `self` with another parameter will cause issues hard to find and fix